Alert.css

```diff
@@ -0,0 +1,71 @@
+.header {
+  text-align: center;
+  color: indianred;
```

Comment:
You can use CSS variables for color. If anything needs to be changed then it can directly be changed in the variables.